### PR TITLE
6280: Call dapi for compare document

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/compare/ui.test.js [hunk 1]
```diff
@@ -15,7 +15,11 @@ fixture('Compare Page')
   .page('http://localhost:1234/healthcheck')
   .afterEach(async (t) => {
     const isDone = nock.isDone();
-    if (!isDone) nock.cleanAll();
+    if (!isDone) {
+      // eslint-disable-next-line no-console
```
Comment (alibombaye) [inline app/pages/compare/ui.test.js:19]: is there a rule is eslint where we can disable this check for ui.test.js?

File: app/apiProvider.js [hunk 2]
```diff
@@ -25,7 +24,8 @@ const endpoints = {
   getPublicSolutionById: options => `${buyingCatalogueApiHost}/api/v1/Solutions/${options.solutionId}/Public`,
   getCapabilities: () => `${buyingCatalogueApiHost}/api/v1/Capabilities`,
   // GET Documents endpoint
-  getDocument: options => `${documentApiHost}/api/v1/Solutions/${options.solutionId}/documents/${options.documentName}`,
+  getSolutionDocument: options => `${documentApiHost}/api/v1/Solutions/${options.solutionId}/documents/${options.documentName}`,
+  getDocument: options => `${documentApiHost}/api/v1/api/v1/documents/${options.documentName}`,
```
Comment (alibombaye) [inline app/apiProvider.js:28]: got `api/v1/` in there twice

File: app/routes.js [hunk 3]
```diff
@@ -76,6 +76,16 @@ export const routes = (authProvider) => {
     res.render('pages/compare/template.njk', addConfig({ context, user: req.user }));
   });
 
+  router.get('/compare/document', async (req, res) => {
+    logger.info('downloading olution comparison document');
```
Comment (alibombaye) [inline app/routes.js:80]: typo `olution`

File: app/routes.js [hunk 4]
```diff
@@ -76,6 +76,16 @@ export const routes = (authProvider) => {
...
+    const response = await getDocument({
+      endpointLocator: 'getDocument',
+      options: { documentName: 'comparesolutions.xlsx' },
```
Comment (alibombaye) [inline app/routes.js:83]: can we rename this to `compare-solutions` we have the covid19 doc hyhenated

Comment (kat1906) [reply app/routes.js:83]: ok but this is what Sam suggested on the json contract

File: app/pages/compare/ui.test.js (hunk 1 above)
Comment (kat1906) [reply app/pages/compare/ui.test.js:19]: I don't think we should turn off this rule for the whole file, that could lead to a lot of console logs in them

File: app/routes.js (hunk 4 above)
Comment (alibombaye) [reply app/routes.js:83]: ok I'll put a comment on the json contract task.

File: app/pages/compare/ui.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/compare/ui.test.js:19]: 👍 ok yeah that's a fair point.

File: app/routes.js (hunk 4 above)
Comment (alibombaye) [reply app/routes.js:83]: Should be fine to rename it to `compare-solutions`

**Pull request merged**